cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,7 +175,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (exception: Exception) {
+            println("Error deleting repository: ${exception.message}")
```

Upon assessing the need for the catch block surrounding the delete operation on the repository, it seems crucial to handle potential exceptions to ensure the stability of the application when operations fail. Since exceptions may occur due to network issues, permissions, or other unforeseen errors when interacting with the Artifactory repository, it is essential to log these issues for diagnostic purposes.<br><br>To address this concern, the existing empty catch block should be replaced with a logging statement. Using `println` fulfills the requirement to log the exception details, including the exception message, enhancing the traceability and debuggability of the code. This modification aligns with the fix strategy and steps to provide transparency for the maintainer about why the exception handling is necessary and what kind of issues occur during the operation. Additionally, this aligns with the language-specific rule to avoid having empty catch blocks.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)